core/Thekdar.js

```diff
@@ -19,8 +21,35 @@ class Thekdar extends EventEmitter {
     this._maxWorkers = 20;
     this._maxTaskPerWorker = 10;
     this._pluggins = [];
+    this._queue = new BetterQueue(this.handleQueue.bind(this), {
+      concurrent: 1,
+      id: 'id',
+      filo: true,
+      maxRetries: 1,
+      retryDelay: 5000,
+      maxTimeout: 15000,
```

Comment:
maybe, make these configurable with defaults?

Reply:
yes, this configurable is default, can be overridden by queue properties from the constructor